scripts/publish.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e -u
+
+yarn build
```

Comment:
[very minor] Maybe move this "yarn build" to the package.json, so the script only publishes.

Reply:
Done